alchemy/tests/test_alchemy.py

```diff
@@ -957,8 +967,10 @@ def overlap_check(reference_system, positions, platform_name=None, precision=Non
 
     # Raise an exception if the error is larger than 3kT.
     MAX_DEVIATION = 3.0 # kT
+    report = "DeltaF = %12.3f +- %12.3f kT (%5d samples, g = %6.1f); du mean %.3f kT stddev %.3f kT" % (DeltaF, dDeltaF, Neff, g, du_n.mean(), du_n.std())
+    logger.info(report)
+    print(report)
```

Comment:
Should we remove the print statement here, if there is already a logger statement?